digitalidentity/yotierror/signed_requests.go

```diff
@@ -0,0 +1,8 @@
+package yotierror
+
+const (
```

Comment:
these aren't used

Reply:
True, but this is a package meant to be used by integrators so I;d like to keep them in if we believe they can add value as consts to compare against.

Reply:
well they would add value if they were used, if they're not used why would they need to compare against them?